llvm/lib/Target/M68k/GISel/M68kCallLowering.h

```diff
@@ -64,11 +64,6 @@ struct M68kIncomingValueHandler : public CallLowering::IncomingValueHandler {
                            ISD::ArgFlagsTy Flags) override;
 };
 
-struct FormalArgHandler : public M68kIncomingValueHandler {
-  FormalArgHandler(MachineIRBuilder &MIRBuilder, MachineRegisterInfo &MRI)
-      : M68kIncomingValueHandler(MIRBuilder, MRI) {}
-};
-
 struct CallReturnHandler : public M68kIncomingValueHandler {
```

Comment:
Would you mind moving this one into the cpp file as well?

Reply:
Done.